microk8s-resources/gitconfig

```diff
@@ -0,0 +1,3 @@
+[safe]
```

Comment:
Can we put this like `microk8s-resources/default-args/git/.gitconfig`?

Reply:
do we want to have it in the repo as a hidden file?